cancer_ai/validator/models.py

```diff
@@ -13,3 +23,28 @@ class CompetitionModel(BaseModel):
 class CompetitionsListModel(BaseModel):
     competitions: List[CompetitionModel]
 
+class DataPackageReference(BaseModel):
```

Comment:
Data Package - everything is data package in programming :) 
method for finding names  - say aloud what does it hold and then shrink it so that is still unique and describes exactly what is it

Reply:
tldr; DatasetReference, though I would prefer DataPackageReference, but not having a strong opinion here

Hmm well thats the process I think i've been following for this one :D but lets try again!
It holds a reference to the organizations hugging face repo/files with the uploaded datasets (or data packages right) - >shrinking< - DatasetReference aka DataPackageReference.
I was leaning towards the second one because it highlights that there are many data packages sent by the org to form an overall dataset.